nwv2-python-wrapper/src/protocol/update.rs

```diff
@@ -0,0 +1,54 @@
+use pyo3::exceptions::*;
```

Comment:
Is that a Rust import?

Reply:
It is! It's importing from this: https://docs.rs/pyo3/latest/pyo3/exceptions/index.html

These are Rust wrappers for some standard Python exception classes like `Exception`, `ValueError`, etc.

Also hi 👋